tests/c_languages/CPPCheckBearTest.py

```diff
@@ -0,0 +1,30 @@
+from bears.c_languages.CPPCheckBear import CPPCheckBear
+from tests.LocalBearTestHelper import verify_local_bear
+
+
+good_file = """
+void f1(struct fred_t *p)
+{
+    int x = p->x;
+    if (p)
+        do_something(x);
+}""".splitlines(keepends=True)
+
+bad_file = """
+#define f(c) { \
+    char *p = new char[10];  \
+    p[c] = 42; \
+}
+int main() {
+    f(100);
+    return 0;
+}""".splitlines(keepends=True)
+
+
+CPPCheckEnableAllFalseTest = verify_local_bear(
+    CPPCheckBear, valid_files=(good_file,), invalid_files=(bad_file,),
+    settings={'enable_all_checks': 'False'})
+
+CPPCheckBearTest = verify_local_bear(CPPCheckBear,
+                                     valid_files=(),
+                                     invalid_files=(good_file, bad_file))
```

Comment:
could you maybe include a `very_good_file` that doesn't get picked up by CppCheckBear at all.